ggu/main.c

```diff
@@ -423,9 +578,15 @@ run_ggu(struct net_config *net_conf,
 	ggu_conf->ggu_src_port = rte_cpu_to_be_16(ggu_conf->ggu_src_port);
 	ggu_conf->ggu_dst_port = rte_cpu_to_be_16(ggu_conf->ggu_dst_port);
 
-	ret = 0;
-	goto out;
+	ret = init_mailbox("ggu_mb", MAILBOX_MAX_ENTRIES,
+		sizeof(struct rte_mbuf *), ggu_conf->lcore_id,
```

Comment:
Shouldn't we use `sizeof(struct ggu_request)` here instead of `sizeof(struct rte_mbuf *)`?